fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/util/Command.java

```diff
@@ -23,10 +25,17 @@ public class Command {
     protected List<String> cmd;
     protected Map<String, String> env;
 
-    Command(List<String> cmd) {
+    Command(List<String> cmd, Map<String,String> additionalEnv){
         this.cmd = cmd;
-        this.env = new HashMap<>();
+        // this.env = new HashMap(System.getenv());
+        this.env = new HashMap();
+        this.env.putAll(additionalEnv);
+    }
 
+    Command(List<String> cmd) {
+        this.cmd = cmd;
+        this.env = new HashMap();
+        // this.env = new HashMap(System.getenv());
```

Comment:
Did you mean to leave this commented out? Will not having the system env cause any problems?